### DISCUSSION TITLE
Strict mode

### DISCUSSION THREAD

**Discussion by fksune** [iSoron/uhabits#1195] [Feature Requests]
Currently we can mark any future day as complete, if this can be restricted by disabling the checkboxes with some toggle it would be great.

**Comment by hiqua** [comment]
Future days? I only see days up to today, I think that was actually a feature request to be able to see in the future, has that been implemented already?

**Reply by iSoron** [reply]
No, that's not implemented yet. It's currently impossible to add checkmarks in future days.

**Comment by herrbrixter** [comment]
Duplicate of #2173
